dist-workspace.toml

```diff
@@ -18,7 +18,7 @@ build-command = [
     '--allow-env="GITHUB_*,GH_*,LINEAR_*"',
     "--allow-read",
     "--allow-write",
-    "--allow-run=git,gh,open,cmd,xdg-open,wslview",
+    "--allow-run=git,gh,open,cmd,xdg-open,wslview,open",
```

Comment:
`open` already exists in all these lists (i had added it to the other PR), i don't think this change is needed